Mathlib/CategoryTheory/Limits/Shapes/Pullback/Square.lean

```diff
@@ -105,6 +106,56 @@ lemma IsPullback.op {sq : Square C} (h : sq.IsPullback) : sq.op.IsPushout :=
 lemma IsPullback.unop {sq : Square Cᵒᵖ} (h : sq.IsPullback) : sq.unop.IsPushout :=
   CategoryTheory.IsPullback.unop h.flip
 
+section
+
+variable
+  {sq₁ : Square (Type v)} (h₁ : sq₁.IsPullback)
+  {sq₂ : Square (Type u)}
+  (e₁ : sq₁.X₁ ≃ sq₂.X₁) (e₂ : sq₁.X₂ ≃ sq₂.X₂)
+  (e₃ : sq₁.X₃ ≃ sq₂.X₃) (e₄ : sq₁.X₄ ≃ sq₂.X₄)
+  (comm₁₂ : e₂ ∘ sq₁.f₁₂ = sq₂.f₁₂ ∘ e₁)
+  (comm₁₃ : e₃ ∘ sq₁.f₁₃ = sq₂.f₁₃ ∘ e₁)
+  (comm₂₄ : e₄ ∘ sq₁.f₂₄ = sq₂.f₂₄ ∘ e₂)
+  (comm₃₄ : e₄ ∘ sq₁.f₃₄ = sq₂.f₃₄ ∘ e₃)
+
+lemma IsPullback.of_equiv : sq₂.IsPullback := by
```

Comment:
What do you think of the following alternative proof going through `uliftFunctor`: https://github.com/leanprover-community/mathlib4/commit/0e493eb7d31439b45039c8e71dc1e490371829fd

Reply:
Thanks for the suggestion. I have used your code.